client/landing/gutenboarding/utils.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * ***Do not reproduce this pattern for Gutenboarding work.***
+ *
+ * FIXME: Replace this with another pattern:
```

Comment:
We're working on site creation in this issue:

https://github.com/Automattic/wp-calypso/issues/38707

It's not done yet, and we might not pick it up again until next week! Feel free to contribute!

We're following a similar pattern to the account creation store

https://github.com/Automattic/wp-calypso/pull/38823